admin_interface/migrations/0027_theme_list_filter_removal_links.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.7 on 2022-10-10 21:46
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('admin_interface', '0026_theme_list_filter_highlight'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='theme',
+            name='list_filter_removal_links',
+            field=models.BooleanField(default=False, verbose_name='quick remove links for active filters at top of sidebar'),
```

Comment:
I like to make new features enabled by default `default=True`, users notice them and report possible issues. What do you think about it?

Reply:
> It makes sense to have default true for a non-intrusive visual improvement like list_filter_highlight, but here it’s a UI change and it’s only useful if you have long list lists of filters in your admins.